src/main/java/com/daitangroup/vehicleImplementations/HondaCivic.java

```diff
@@ -0,0 +1,44 @@
+package com.daitangroup.vehicleImplementations;
+
+import com.daitangroup.vehiclesTypes.Car;
+
+import java.util.Objects;
+
+public class HondaCivic extends Car {
+
+
+    private int horsePower = 150;
+    private String engineDescription = "2.0 flex fuel";
+    private boolean parkAssist = true;
+
+
+    public HondaCivic() {
+        super(50, 6, 525, 4, 5);
+    }
+
+
+    @Override
+    public String toString() {
+        return "HondaCivic{" +
+                "horsePower=" + horsePower +
+                ", engineDescription='" + engineDescription + '\'' +
+                ", parkAssist=" + parkAssist +
+                '}' + super.toString();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if ( !(o instanceof HondaCivic) ) return false;
+        if (!super.equals(o)) return false;
+        HondaCivic that = (HondaCivic) o;
+        return horsePower == that.horsePower &&
+                parkAssist == that.parkAssist &&
+                Objects.equals(engineDescription, that.engineDescription);
```

Comment:
I think it would be better to use the Equals from the String class.

Reply:
I disagree. Objects.equals is safer and String.equals gains you little besides readability.